public/src/components/game.js

```diff
@@ -69,7 +69,12 @@ export default React.createClass({
     let startControls = d.div({},
       d.div({}, `Format: ${App.state.format}`),
       LBox('bots', 'bots'),
-      LBox('timer', 'timer'),
+      d.div({}, d.input({
+        min: 0,
+        max: 60,
+        type: 'number',
+        valueLink: App.link('timer'),
+        }), ' second timer'),
```

Comment:
You should wrap the text and its `input` in a `label`, so that one can click the text and it focuses the input. It's also probably good for accessibility.